### ISSUE TITLE
crio-shutdown.service fails with Exec format error / Executable path is not absolute

### ISSUE THREAD

**Issue by adelton** [kubernetes-sigs/cri-o#1413]
**Description**

I've tried to use the `make install.systemd` target. It installs

```
install -Z -D -m 644 contrib/systemd/crio.service /usr/local/lib/systemd/system/crio.service
ln -sf crio.service /usr/local/lib/systemd/system/cri-o.service
install -Z -D -m 644 contrib/systemd/crio-shutdown.service /usr/local/lib/systemd/system/crio-shutdown.service
```

However, `systemctl status crio-shutdown.service` complains about

```
● crio-shutdown.service - Shutdown CRIO containers before shutting down the system
   Loaded: error (Reason: Exec format error)
   Active: inactive (dead)
     Docs: man:crio(8)

Mar 03 05:51:08 test.example.com systemd[1]: /usr/local/lib/systemd/system/crio-shutdown.service:10: Executable path is not absolute: mkdir -p /var/lib/crio; touch /var/lib/crio/crio.shutdown
```

**Steps to reproduce the issue:**
1. `make`
2. `make install.systemd`
3. `systemctl daemon-reload`
4. `systemctl status crio-shutdown`

**Describe the results you received:**

```
● crio-shutdown.service - Shutdown CRIO containers before shutting down the system
   Loaded: error (Reason: Exec format error)
   Active: inactive (dead)
     Docs: man:crio(8)

Mar 03 05:51:08 test.example.com systemd[1]: /usr/local/lib/systemd/system/crio-shutdown.service:10: Executable path is not absolute: mkdir -p /var/lib/crio; touch /var/lib/crio/crio.shutdown
```

**Describe the results you expected:**

No error.

**Additional information you deem important (e.g. issue happens only occasionally):**

Since `git grep crio-shutdown` does not find any documentation for that service, maybe it is not needed and can be removed completely?

**Output of `crio --version`:**

```
crio version 1.10.0-dev
commit: "0b736bb43fb9f6ffa29b477be9e292bec3686ef4"
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by rhatdan** [comment]
The idea of the cri-shutdown service is to shut the containers down on system halt/reboot.

If you shutdown the crio service, the containers keep running, if you are shutting down the system this service is supposed to tell the crio service to shutdown all the containers so then can get a clean shutdown, as opposed to systemd killing all the processes.

**Comment by umohnani8** [comment]
Tried this with 1.14.0 and don't see an error. @adelton please reopen if you are still seeing this issue. Closing.

**Comment by adelton** [comment]
Well, I assume this was fixed with 309667f5158e7e13a47f57d24f6b69ee97aed871.
